airbyte-ci/connectors/connectors_insights/pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "connectors-insights"
+version = "0.1.0"
+description = ""
+authors = ["Airbyte <contact@airbyte.io>"]
+readme = "README.md"
+packages = [{ include = "connectors_insights", from = "src" }]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+dagger-io = "0.9.6"
+click = "^8.1.7"
+pydantic = "^1.9"
```

Comment:
Be ready to jump to 2.0 as the CDK will upgrade. 

Maybe a side quest to upgrade all airbyte-ci projects to use pydantic v2. What's blocking us?

Reply:
I can do it in a different PR. I think we have some internal packages depending on the CDK which explains why they're also on pydantic 1.9.